core/services/log/orm.go

```diff
@@ -43,6 +42,8 @@ func (o *orm) UpsertLog(log types.Log) error {
 		copy(x, topic[:])
 		topics[i] = x
 	}
+	data := []byte{}
```

Comment:
The log data is not nullable. It appears pq will take an nil byte slice and convert it into a empty bytea in postgres, pgx will not do that.